themes/base/theme.css

```diff
@@ -80,18 +89,31 @@
 .ui-state-focus a,
 .ui-state-focus a:hover,
 .ui-state-focus a:link,
-.ui-state-focus a:visited {
+.ui-state-focus a:visited,
+a.ui-button:hover,
+a.ui-button:focus {
 	color: #2b2b2b/*{fcHover}*/;
 	text-decoration: none;
 }
+.ui-visual-focus {
+	box-shadow: 0 0 3px 1px rgb(94, 158, 214);
```

Comment:
Let's add a comment that describes why this isn't configurable through ThemeRoller (just added). Maybe indicating that we consider exposing it later, if the concept works out.